bin/cli.js

```diff
@@ -3,17 +3,17 @@
 'use strict';
 
 const minimist = require('minimist');
-const _ = require('lodash');
+const camelCase = require('lodash.camelcase');
 const checkDependencies = require('../lib/check-dependencies');
 
 const argv = minimist(process.argv.slice(2));
 
 // camelCase the options
-for (const key in argv) {
+Object.keys(argv).forEach(key => {
```

Comment:
It is about enumerable properties being shadowed by non-enumerables and skipped. That would be a concern, mainly, if we would need prototype chain, but we don't. That's why I purpose this change: using `for in` is kinda overkill here.